plugins/utils/neogen.nix

```diff
@@ -112,53 +112,53 @@ in {
           - `"vsnip"` (https://github.com/hrsh7th/vim-vsnip)
       '';
 
-      placeholderHighligt = helpers.defaultNullOpts.mkStr "DiagnosticHint" ''
+      placeholderHighlight = helpers.defaultNullOpts.mkStr "DiagnosticHint" ''
```

Comment:
We  need to introduce a warning that this option was renamed, even if it was a  typo :/ See how other renames are done